src/libmongoc/src/mongoc/mongoc-client.c

```diff
@@ -1745,6 +1761,21 @@ _mongoc_client_retryable_write_command_with_stream (
       mongoc_server_stream_cleanup (retry_server_stream);
    }
 
+   // If a retry attempt fails with an error labeled NoWritesPerformed,
+   // drivers MUST return the original error.
+   if (original_error.set && !ret &&
```

Comment:
```suggestion
   if (original_error.set &&
```
Suggestion to add `!ret` was mistaken as `ok: 1` may be present alongside error labels.